.github/workflows/main.yml

```diff
@@ -62,11 +62,7 @@ jobs:
       - name: Set up MSVC
         uses: microsoft/setup-msbuild@v2
       - name: Build and register the PyCOMTest server dll
-        run: |
-          cd com/TestSources/PyCOMTest
-          msbuild .\PyCOMTest.sln -property:Configuration=Release
-          cd x64/Release
-          regsvr32 .\PyCOMTest.dll
+        run: com/TestSources/PyCOMTest/buildAndRegister.bat /s
```

Comment:
isn't this likely to work even if the build or regsvr fails?

Reply:
This step may pass. But the test would still fail if the registration fails.

I added this param because otherwise the step hangs forever, with the script waiting for confirmation to finish.

Reply:
I understand the `/s`, but don't really see the benefit of the new .bat file, especially given it might be misleading when the build or registration fails.

Reply:
Just meant as a helper script to build the required project for testing. And avoid duplication.
I could add the lines to `com/win32com/test/readme.txt` instead.

The helper message when the test finds the server isn't registered (what's assigned to `importMsg`) should probably point to that readme explicitly.

When we were debugging PyComTests is took me a while to find it even existed.